creator-node/src/ImageProcessingQueue.ts

```diff
@@ -1,19 +1,20 @@
-const { Queue, QueueEvents, Worker } = require('bullmq')
-const path = require('path')
-const os = require('os')
+import type { PrometheusRegistry } from './services/prometheusMonitoring/prometheusRegistry'
 
-const config = require('./config')
-const { logger: genericLogger } = require('./logging')
-const { clusterUtils } = require('./utils')
-const resizeImage = require('./resizeImage')
+import { Queue, QueueEvents, Worker } from 'bullmq'
+import path from 'path'
+import os from 'os'
+
+import config from './config'
+import { logger as genericLogger } from './logging'
+import { clusterUtils } from './utils'
 
 const imageProcessingMaxConcurrency = config.get(
   'imageProcessingMaxConcurrency'
 )
 
-const PROCESS_NAMES = Object.freeze({
-  resizeImage: 'resizeImage'
-})
+enum ProcessNames {
```

Comment:
didn't you post in slack that const objects > enums in TS?
https://twitter.com/justinfagnani/status/1570475857466261505?s=20&t=89gQIv4_Iu3PMRNQQkY9Hw

Reply:
Yup - it's a little weird in this case cause `resizeImage` is the _only_ process name lol so I was thinking about getting rid of the enum entirely. I ended up just having it as a enum because that was the least likely thing to cause anything to break

Reply:
never mind, I fixed it